components/tab-switcher.jsx

```diff
@@ -0,0 +1,81 @@
+// Note: source code of this component is pretty much copied directly from https://github.com/mozilla/science.mozilla.org
```

Comment:
Yea I really wish we could make use of `mofo-ui`. Oh well! We'll make it happen gradually! :dancing_men: 👯